### ISSUE TITLE
ACF Pro Support

### ISSUE THREAD

**Issue by sissibieber** [airesvsg/acf-to-rest-api#39]
Hello,
is there anything planned on ACF Pro support for the REST API?
Thank you,
Sissi

**Comment by airesvsg** [comment]
Hi @sissibieber,
the plugin supports ACF Pro.
Thanks

**Comment by sissibieber** [comment]
Hello, sorry to open this again, but for PUT/POST to users/<id> it did not work (I don't have the errors right now, but if you say that GET/PUT/POST should work with ACF Pro as well, I will switch back to PRO again)? Or are there just limitations for "users"?
Thank you, Sissi

**Comment by airesvsg** [comment]
Hi @sissibieber,
please see this issue: https://github.com/airesvsg/acf-to-rest-api-example/issues/1
Thanks

**Comment by sissibieber** [comment]
Ok, thank you - it does work - I had the the RULES set to AND instead of OR, which makes no sense. Also I changed the "important" rule current user is logged in to current user is "in frontend" which seems to work as well for POST/PUT via the API. In the other case I had the field group I created on every page in the backend which was a bit irritating, now it only is shown on the user form.

Thank you for that great development!
